.claude/skills/review-pr/SKILL.md

```
@@ -1,281 +1,97 @@
+- Check for N+1 queries (missing fetch join or `@BatchSize`)
+- No sensitive data exposed in responses
+
+## Step 3: Output format
 
-GitHub CLI로 PR의 기본 정보와 변경사항을 파악합니다.
-
-```bash
-gh pr view <번호> -R solid-connection/solid-connect-server      # PR 기본 정보 조회
-gh pr diff <번호> -R solid-connection/solid-connect-server      # 변경된 파일과 diff 확인
-gh pr checks <번호> -R solid-connection/solid-connect-server    # CI/CD 상태 확인
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)